Update install.md (#541)
replicatedhq/kots.io: content/kots-cli/admin-console/upgrade.md

| `-n, --namespace` | string | the namespace where the admin console is running _(default "default")_         |
| `--wait-duration`  | string | timeout out to be used while waiting for individual components to be ready.  must be in [Go duration](https://pkg.go.dev/time#ParseDuration) format (eg: 10s, 2m). |
| `--with-minio`    | bool   | when set, kots will deploy a local minio instance for storage _(default true)_ |

> | `--ensure-rbac`          | bool   | when set, kots will skip RBAC configuration at upgrade time. (default false) if a role spec is needed, use the [generate-manifests](/kots-cli/admin-console/generate-manifests/) command.                                                         |

Reviewer: ```suggestion
| `--ensure-rbac`          | bool   | when set, kots will skip RBAC configuration at upgrade time (default false). if custom role specs are needed, use the [generate-manifests](/kots-cli/admin-console/generate-manifests/) command.                                                         |
```
